issue_template.md

```diff
@@ -6,9 +6,12 @@
 
 ## Steps to reproduce
 
-> _Include configurations used in your: <br>
+> 1. _Include configurations used in your: <br>
 > (1) .codeclimate.yml <br>
-> (2) config files for the oss analysis tool._
+> (2) config files for the oss analysis tool._<br>
+
+> 2. _If possible, use our CLI with [the debug option](https://github.com/codeclimate/codeclimate#environment-variables) to provide additional info, including any stderr produced by the engine._ `CODECLIMATE_DEBUG=1 codeclimate analyze`
```

Comment:
If people use more than one engine this can be a *lot* of output. Maybe we should specify running just `eslint` here so there'll be less output to dig through?

`CODECLIMATE_DEBUG=1 codeclimate analyze -e eslint`